package.json

```diff
@@ -239,15 +239,15 @@
       "type": "object",
       "title": "Docker configuration options",
       "properties": {
-        "docker.defaultRegistry": {
-          "type": "string",
-          "default": "",
-          "description": "Default registry when tagging an image, empty string will target Docker Hub when pushing."
-        },
         "docker.defaultRegistryPath": {
           "type": "string",
           "default": "",
-          "description": "Path within registry to push to."
+          "description": "Default registry and path when tagging an image, empty string will target Docker Hub when pushing"
+        },
+        "docker.askToSavePrefix": {
```

Comment:
Why adding another concept of prefix here?  We already have the concept of defaultRegistryPath, so how about askToSaveRegistryPath?